### DISCUSSION TITLE
How to get .vrm model face's coordinates?

### DISCUSSION THREAD

**Discussion by ALABOET** [pixiv/three-vrm#1117] [Help] [answered]
Hello there! I want to zoom on the face of my model, however right now i just hardcoded it which is bad, because different models may have different sizes, heights. So the question is: how can I get vrm model face's coordinates so i could know where to zoom to? Thank you!

**Comment by 0b5vr** [comment, marked as answer]
You would want to choose either of them:

- [`vrm.humanoid.getNormalizedBoneNode('head').getWorldPosition()`](https://pixiv.github.io/three-vrm/packages/three-vrm/docs/classes/VRMHumanoid.html#getNormalizedBoneNode) gives you the position of the head bone.
- [`vrm.lookAt.getLookAtWorldPosition()`](https://pixiv.github.io/three-vrm/packages/three-vrm/docs/classes/VRMLookAt.html#getLookAtWorldPosition) gives you the position of the lookAt (i.e. their view position, the center of the eyes)

While you can get the position of the head using these methods, you can't retrieve the size of the head unfortunately. You would want to do the hardcoding for this.

**Reply by ALABOET** [reply]
Thank you so much!
